### ISSUE TITLE
Refont domain registration page step 1

### ISSUE THREAD

**Issue by Kevils** [starknet-id/app.starknet.id#951]
## Description 📹

Current one : https://app.starknet.id/register/kevil.stark
<img width="1440" alt="image" src="https://github.com/user-attachments/assets/98c7cbb7-7176-4574-91a1-2e9e3f3e3d55" />

New design :
<img width="694" alt="image" src="https://github.com/user-attachments/assets/946af0a3-ce70-4c5d-89d2-251faa799801" />

Figma file : https://www.figma.com/design/S1UKYgWewNqNHZFAaBUilG/%F0%9F%8F%9D%EF%B8%8F-Starknet-ID?node-id=4111-49838&t=wEy6O4uIMpUixad7-1

**New redesigned domain registration step 1. Add the possibility to close or back with “close icon”**

## Proposed Actions 🛠️

Here’s a checklist of actions to follow for resolving this issue:

1. **Fork and Create Branch**:  
   Fork the repository and create a new branch using the issue number:
   ```bash
   git checkout -b fix-[issue-number]
   ```

2. **Implement Changes**:  
	[Insert Code snippet if needed with a mardown todo list]

3. **Run Tests and Commit Changes**:  
   Make sure your changes don't break existing functionality and commit with a clear message:
   ```bash
   git commit -m "Fix: [Short description of the fix]"
   ```

## Required 📋

To keep our workflow smooth, please make sure you follow these guidelines:

- **Assignment**: Don't create a pull request if you weren’t assigned to this issue.
- **Timeframe**: Complete the task within **3 business days**.
- **Closing the Issue**: In your PR description, close the issue by writing `Close #[issue_id]`.
- **Review Process**:
  - Once you've submitted your PR, change the label to **"ready for review"**.
  - If changes are requested, address them and then update the label back to **"ready for review"** once done.
- **Testing**: Test your PR locally before pushing, and verify that tests and build are working after pushing.

Thank you for your contribution 🙏

⚠️ WARNING: Failure to follow the requirements above may result in being added to the OnlyDust blacklist, affecting your ability to receive future rewards.

**Comment by ryzen-xp** [comment]
I'd love to give this a go.

**Comment by Amarjeet325** [comment]
Can I contribute to this one?

**Comment by SudiptaPaul-31** [comment]
Can I work on this issue? @Kevils
I am a fullstack and blockchain developer with experience for more than 1.5 years and would love to work on this issue

ETA-1Day

**Comment by Nityam573** [comment]
Could I take a shot at this?

**Comment by mimisavage** [comment]
Can I tackle this one?

**Comment by caxtonacollins** [comment]
I'd love to work on and convert this new redesigned domain registration design;
I will also add the close icon for easy navigation. ETA 3 working days.

**Comment by chiscookeke11** [comment]
Can I attempt this issue?

**Comment by Elite-tch** [comment]
I'd like to take this issue.

**Comment by covenantebube** [comment]
Hi, My name is covenant ebube. I am a MERN stack developer with years of experience solving real-world problems using Javascript, React and Typescript. I am a first time contributor on this platform and i am very confident that i can solve this task within the specified time frame. Can you please assign this task?

**Comment by caxtonacollins** [comment]
@Kevils, please am having issues running the project

![Image](https://github.com/user-attachments/assets/09ffb6a3-2263-4b05-85a2-5a9e95b5477c)

**Comment by Marchand-Nicolas** [comment]
Hello @caxtonacollins  are you working on the issue ? If I don't have any sign of life I must unassign you

**Comment by caxtonacollins** [comment]
> Hello @caxtonacollins  are you working on the issue ? If I don't have any sign of life I must unassign you
Yes i am ser, will create a PR to close the issue in few hours..

**Comment by Marchand-Nicolas** [comment]
> > Hello [@caxtonacollins](https://github.com/caxtonacollins)  are you working on the issue ? If I don't have any sign of life I must unassign you

> Yes i am ser, will create a PR to close the issue in few hours..

Incredible thanks
